domains/kunalx.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "For personal portfolio website.",
+    "repo": "https://github.com/KunalScriptz/register",
+    "owner": {
+        "username": "KunalScriptz",
+        "email": "kunal1520018@gmail.com"
+    },
+    "record": {
+        "CNAME": "KunalScriptz.github.io"
```

website seems to be a template
```suggestion
        "CNAME": "kunalscriptz.github.io"
```